src/packages/sdk/src/utils/load.ts

```diff
@@ -0,0 +1,8 @@
+/** This is a wrapper around `require` */
```

Comment:
Would be good to say why?
I think because of https://github.com/vercel/nft?

Reply:
I'm not 100% sure, will ask @millsp when he gets in

Reply:
This is to avoid `eval` and hide `require` away from bundlers

Reply:
While the try catch is just to silence esbuild, I think.

Reply:
```suggestion
/** 
 ** This is a wrapper around `require` 
 ** This is to avoid eval and hide require away from bundlers
*/
```